lib/bundler/cli/install.rb

```diff
@@ -185,5 +181,13 @@ def print_post_install_message(name, msg)
       Bundler.ui.confirm "Post-install message from #{name}:"
       Bundler.ui.info msg
     end
+
+    def check_for_options_conflicts
```

Comment:
This would also result in the output being:
```
$ bundle install  --system --deployment
You have specified both --deployment as well as --system. Please choose.
Bundler::InstallError #<-- in red
```